scripts/build.sh

```diff
@@ -167,11 +222,13 @@ case "$product-$method-$platform" in
   Firestore-cmake-macOS)
     test -d build || mkdir build
     echo "Preparing cmake build ..."
-    (cd build; cmake ..)
+    set +u
```

Comment:
Better than disabling undefined variable checking to deal with a possibly empty array is to find some way to avoid the empty array.

Perhaps take the opportunity to enable warnings in the cmake run? Some good candidates might be:
  * `-Wdeprecated`
  * `--warn-uninitialized`
  * etc.